utils/appliance/__init__.py

```diff
@@ -2172,6 +2174,48 @@ def configure_appliance_for_openldap_ext_auth(self, appliance_fqdn):
             'setenforce 0 && systemctl restart sssd && systemctl restart httpd')
         self.wait_for_web_ui()
 
+    @logger_wrap("Configuring VM Console: {}")
+    def configure_vm_console_cert(self, log_callback=None):
+        """This method generates a self signed SSL cert and installs it
+           in the miq/vmdb/certs dir.   This cert will be used by the
+           HTML 5 VM Console feature.  Note evmserverd needs to be restarted
+           after running this.
+        """
+        log_callback('Installing SSL certificate')
+        cert = conf.cfme_data['vm_console']['cert']
```

Comment:
so in this case, if 'vm_console' or 'cert' don't exist, you'll get a KeyError exception.

If you used conf.cfme_data['vm_console'].get('cert', None), you won't get an exception when 'cert' DNE, you'll get None as the value.  Then you can check for None and log a descriptive error message + re-raise the KeyError or raise a more specific exception. 

Its an optional change, just makes things a bit cleaner and clearer when the YAML file is missing a map.